test/mongoose_push_test.exs

```diff
@@ -182,6 +184,7 @@ defmodule MongoosePushTest do
       assert notification.alert[:tag] == fcm_data["tag"]
       assert notification.alert[:sound] == fcm_data["sound"]
       assert notification[:data] == fcm_custom
+      assert notification[:time_to_live] == fcm_data["time_to_live"]
```

Comment:
According to FCM documentation, `time_to_live` should be at the top level, among things like `priority`. This means that this check should be: 

```
assert notification[:time_to_live] == fcm_request["request_data"]["time_to_live"]
```